frontends/p4/fromv1.0/converters.cpp

```diff
@@ -1166,7 +1146,7 @@ class CheckIfMultiEntryPoint: public Inspector {
 // does not use the @packet_entry pragma.
 class InsertCompilerGeneratedStartState: public Transform {
     ProgramStructure* structure;
-    IR::IndexedVector<IR::Node>        allTypeDecls;
+    IR::Vector<IR::Node>               allTypeDecls;
```

Comment:
Why change this from a more efficent IndexedVector to a less efficient Vector?  Particularly as this ends up being the global scope, having to do a linear search through the vector could get expensive for large programs?